test/test_jit.py

```diff
@@ -8168,7 +8168,7 @@ def __init__(self):
 
         with self.assertRaisesRegex(
                 TypeError,
-                "'Linear' object for attribute 'invalid' is not a valid constant"):
```

Comment:
curious, why removing the leading '? :-)

Reply:
Because it's not torch.nn.quantized.Linear but I was lazy to copy-paste it :)